Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```
@@ -395,7 +395,7 @@
     access: [ [ "Command" ] ]
   - type: StorageFill
     contents:
-    - id: WeaponRevolverDeckard
+    - id: DeckardBoxFilled
```

Heya, I know you're still working on this, but don't forget about the licensing comments. Make sure to add '# Modified by Ronstation contributor(s), therefore this file is licensed as MIT sublicensed with AGPL-v3.0.' to the top of the file, and have comments where stuff is modified from upstream, like so

```suggestion
    - id: DeckardBoxFilled # Ronstation - modification
```